src/lib/storage/fake_device_factory.rb

```diff
@@ -174,7 +174,7 @@ def fixup_param(name, param)
       #
       # FIXME: this method is too complex. It offends three different cops
       # related to complexity.
-      # rubocop:disable Metrics/PerceivedComplexity, Metrics/AbcSize, Metrics/CyclomaticComplexity
+      # rubocop:disable Metrics/PerceivedComplexity, Metrics/CyclomaticComplexity
```

Comment:
Ah, then no. :wink:  I didn't check. I just saw 'disable' and assumed the same problem-